packages/@coorpacademy-components/src/atom/select-tile/index.js

```diff
@@ -0,0 +1,79 @@
+import React, {useCallback} from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import FaIcon from '../icon';
+
+// eslint-disable-next-line css-modules/no-unused-class
+import style from './style.css';
+
+const iconColor = '#0061FF';
+const getButtonContent = (faIcon, selectionMode) => {
+  const selected = selectionMode === 'single' || selectionMode === 'multi';
+  const checkIconNameMap = {
+    single: 'circle-check',
+    multi: 'square-check'
+  };
+  return (
+    <div className={style.contentWrapper}>
+      {selected ? (
```

Comment:
at first sight it's a little hard to understand, why there is 2 FaIcon,
maybe add a comment or split into differents function, just to understand this is for the checkbox and that for the icon itself